CoreImageProject/CoreImageProject/Main/ViewController.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  ViewController.swift
+//  CoreImageProject
+//
+//  Created by Kenny on 6/1/20.
+//  Copyright © 2020 Hazy Studios. All rights reserved.
+//
+
+import UIKit
+import CoreImage
+import CoreImage.CIFilterBuiltins
+
+class ViewController: UIViewController {
+    // MARK: - Outlets -
+    //Image View
+    @IBOutlet private var imageView: UIImageView!
+    //Stack Views
+    @IBOutlet weak var colorStack: UIStackView!
+    @IBOutlet weak var colorStack2: UIStackView!
+    @IBOutlet weak var sliderStack1: UIStackView!
+    @IBOutlet weak var sliderStack2: UIStackView!
+    @IBOutlet weak var sliderStack3: UIStackView!
+    
+
+    //Slider Labels
+    @IBOutlet weak var slider1Label: UILabel!
+    @IBOutlet weak var slider2Label: UILabel!
+    @IBOutlet weak var slider3Label: UILabel!
+    //Sliders
+    @IBOutlet weak var slider1: UISlider!
+    @IBOutlet weak var slider2: UISlider!
+    @IBOutlet weak var slider3: UISlider!
+
+
+    // MARK: - Properties -
+    private var originalImage: UIImage?
+    ///set the filter's inputImage
+    private var inputImage: CIImage? {
+        guard let originalImage = originalImage,
+            let cgImage = originalImage.cgImage else { return nil }
+            let ciImage = CIImage(cgImage: cgImage)
+        return ciImage
+    }
+    private let context = CIContext(options: nil)
+    private var filter: CIFilter = .gaussianBlur() {
```

Comment:
I'd recommend more consistent spacing between elements. I prefer 1 space between groups.

At some point I might also stop restating comments for certain sections.

```swift
// MARK: - Outlets -

@IBOutlet private var imageView: UIImageView!
@IBOutlet weak var colorStack: UIStackView!
@IBOutlet weak var colorStack2: UIStackView!
@IBOutlet weak var sliderStack1: UIStackView!
@IBOutlet weak var sliderStack2: UIStackView!
@IBOutlet weak var sliderStack3: UIStackView!

@IBOutlet weak var slider1Label: UILabel!
@IBOutlet weak var slider2Label: UILabel!
@IBOutlet weak var slider3Label: UILabel!

@IBOutlet weak var slider1: UISlider!
@IBOutlet weak var slider2: UISlider!
@IBOutlet weak var slider3: UISlider!

// MARK: - Properties -
private var originalImage: UIImage?
```